selfdrive/controls/lib/drive_helpers.py

```diff
@@ -1,16 +1,25 @@
 import math
 from cereal import car
 from common.numpy_fast import clip, interp
+from common.params import Params
 from common.realtime import DT_MDL
 from selfdrive.config import Conversions as CV
 from selfdrive.modeld.constants import T_IDXS
 
+IS_METRIC = Params().get_bool("IsMetric")
```

Comment:
this needs to update as the toggle is flipped

Reply:
Gonna need some help. I'm not sure how I would go about doing that